packages/test-apps/nextjs/src/utils/earlyPreload.ts

```diff
@@ -0,0 +1,101 @@
+import type { Experience } from '@contentful/experiences-sdk-react';
+import type { ContentfulClientApi, Asset, Entry } from 'contentful';
+import { inMemoryEntities } from '@contentful/experiences-sdk-react';
+import {
+  extractReferencesFromEntriesAsIds,
+  extractLeafLinksReferencedFromExperience,
+} from '@contentful/experiences-core';
```

Comment:
```suggestion
import {
  extractReferencesFromEntriesAsIds,
  extractLeafLinksReferencedFromExperience,
} from '@contentful/experiences-sdk-react';
```
We must re-export this in the `index.ts` file of our SDK for public availability

Reply:
I see you already exported it fmor `index.ts`, so closing it 👍🏻